blocks/search-results/search-results.js

```diff
@@ -0,0 +1,42 @@
+import { createCardBlock } from '../card/card.js';
+import { loadBlock } from '../../scripts/lib-franklin.js';
+
+export async function searchResults(params, jsonData) {
```

Comment:
This method does not need to be `async`

Reply:
we do have 2 await functions here and hence async will be required. Pl advise - 

export async function searchResults(params) {
  const allData = await fetch(`${window.location.origin}/articles.json?sheet=full`);
  console.log(allData);
  const jsonData = await allData.json();
  return jsonData.data.filter((entry) => (
    entry.title
    + entry.content
    + entry.path
    + entry.authors
    + entry.collections
    + entry.section
    + entry.categories
  )
    .toLowerCase()
    .includes(params.toLowerCase()));
}